src/gdcdictionary/schemas/aligned_reads.yaml

```diff
@@ -222,6 +222,24 @@ properties:
       - "_terms.yaml#/tumor_purity/common"
     type: number
 
+  wgs_coverage:
+    description: >-
```

Comment:
_terms file has wgs_coverage, ref?

Reply:
Fixed to point to ref.  Will do for other nodes too.